FlowCryptCommon/Extensions/ErrorExtensions.swift

```diff
@@ -14,7 +14,9 @@ public extension Error {
         case let self as CustomStringConvertible:
             return String(describing: self)
         default:
-            return localizedDescription
+            return localizedDescription.trimmingCharacters(
+                in: CharacterSet(charactersIn: "\"") // remove quotes from error message
+            )
```

Comment:
I imagine the quotes could sometimes be meaningful in error messages, like quoting some wrong input. Which may eg be empty and without quotes, that would be difficult to describe

Reply:
It removes quotes only in the beginning and end of the string, so `"some client err"` becomes `some client err`.
I added this trimming because quotes were automatically added to the error message from `wrong-text` test.

There can be an issue if we have quotes only on 1 side, like `wrong format: "some text"`, it'll be transformed into `wrong format: "some text`.

I can leave as it was before - without trimming, or improve current logic to better handle such cases, what do you prefer?

Reply:
I think we can do without trimming. Thanks!